packages/ketcher-macromolecules/src/components/monomerLibrary/RnaBuilder/RnaAccordion/RnaAccordion.tsx

```diff
@@ -91,11 +94,48 @@ export const RnaAccordion = ({ libraryName, duplicatePreset, editPreset }) => {
 
   const [expandedAccordion, setExpandedAccordion] =
     useState<RnaBuilderItem | null>(activeRnaBuilderItem);
+  const [newPreset, setNewPreset] = useState(activePreset);
 
+  useEffect(() => {
```

Comment:
Is this useEffect needed here? There is the same in 
`RnaEditorExpanded.tsx` component, isn't it enough to cover this case? If not, I'd suggest to move this effect up components tree, put it to some common parent component, probably, `RnaBuilder.tsx`

Reply:
Yeap, you right it's not needed here, removed